changelogs/add-experimental_tasklist_toggle

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: Add
+
+Add expandable tasklist item
```

Comment:
Not sure if this will require a global changelog, just a package changelog is fine.

Reply:
Good idea! I changed it in the commit 7eb3b7a